SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -270,6 +270,21 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Cloud Connection")
 	bool bPreventClientCloudDeploymentAutoConnect;
 
+	/*
+	 * -- EXPERIMENTAL --
+	 * This will enable event tracing for the Unreal client/worker.
+	 */
+	UPROPERTY(EditAnywhere, Config, Category = "Event Tracing")
+	bool bEventTracingEnabled;
+
+	/*
+	 * -- EXPERIMENTAL --
+	 * Same as bEventTracingEnabled, but used if WITH_EDITOR is defined.
+	 */
+	UPROPERTY(EditAnywhere, Config, Category = "Event Tracing")
+	bool bEventTracingEnabledWithEditor;
+
+	friend class AEventTracingSettingsOverride;
```

Comment:
allow this test class to access the settings directly so it can test them independently